### PR TITLE
update polls and user apps status

### PR DESCRIPTION
**Pull request by MCausc78**
dsharpplus
netcord
jda
discordeno
eris
nextcord
disgo
dfx

### CODE REVIEW

**Comment by mlnrDev** [comment]
could you please update the polls status for disgo as well? ty: https://github.com/disgoorg/disgo/pull/346 https://github.com/disgoorg/disgo/releases/tag/v0.18.3

**Comment by tim-smart** [comment]
Could you add dfx too? https://github.com/tim-smart/dfx/blob/main/src/types.ts#L1204

Saves opening another PR :)

File: libs.ts [hunk 1]
```diff
@@ -683,24 +680,15 @@ export const libs: Lib[] = [
 		slashCommands: 'Yes',
 		buttons: 'Yes',
 		selectMenus: 'Yes',
-		threads: {
-			text: 'Dev Version',
-			url: 'https://github.com/Discord4J/Discord4J/issues/958'
-		},
+		threads: 'Yes',
```
Comment (advaith1) [inline libs.ts:683]: looks like threads are only available in the [v3.3.0-M2](https://github.com/Discord4J/Discord4J/releases/tag/3.3.0-M2) prerelease and not in the released version v3.2.6, so this change should be reverted

File: libs.ts [hunk 2]
```diff
@@ -860,7 +848,10 @@ export const libs: Lib[] = [
 			text: 'Has a PR',
 			url: 'https://github.com/discordeno/discordeno/pull/3502'
 		},
-		polls: 'No'
+		polls: {
+			text: 'Has a PR',
+			url: 'https://github.com/discordeno/discordeno/pull/3542'
+		}
```
Comment (advaith1) [inline libs.ts:854]: looks like this has since been merged

```suggestion
		polls: 'Dev Version'
```

**Review by advaith1 — CHANGES_REQUESTED** [review]
thanks for the PR! some notes:

**Pull request merged**